packages/block-library/src/video/transforms.js

```diff
@@ -92,6 +92,10 @@ const transforms = {
 					poster: videoElement.getAttribute( 'poster' ) || undefined,
 					src: videoElement.getAttribute( 'src' ) || undefined,
 				};
+				if ( isBlobURL( attributes.url ) ) {
+					attributes.blob = attributes.url;
+					delete attributes.url;
+				}
```

Comment:
I think this should be `attributes.src` instead of `attributes.url`.

Reply:
Thanks for fixing it